src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -112,17 +107,12 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
           });
           if (application && http) {
             const newWorkspaceId = result.result.id;
-            const useCaseId = getFirstUseCaseOfFeatureConfigs(attributes.features);
-            const useCaseLandingAppId = availableUseCases?.find(({ id }) => useCaseId === id)
-              ?.features[0].id;
             // Redirect page after one second, leave one second time to show create successful toast.
             window.setTimeout(() => {
-              window.location.href = formatUrlWithWorkspaceId(
-                application.getUrlForApp(useCaseLandingAppId || WORKSPACE_DETAIL_APP_ID, {
-                  absolute: true,
-                }),
+              navigateToWorkspaceDetail(
+                { application, http },
                 newWorkspaceId,
-                http.basePath
+                DetailTab.Collaborators
```

Comment:
do we need to check permission enabled?

Reply:
Good point, I think we should check this. Let me add a fallback logic here.